### ISSUE TITLE
org-journal loops when carrying over items with a properties drawer

### ISSUE THREAD

**Issue by brabalan** [bastibe/org-journal#256]
**Describe the bug**
I updated org-journal and it's looping when carrying items over. I think this is due to the presence of a properties drawer.

**To Reproduce**
Have a previous task like this:

```org
* TODO Resserrer vélo elliptique
:PROPERTIES:
:CREATED:  [2020-05-29 Fri 08:23]
:LAST_REVIEW: 2020-05-29 Fri
:NEXT_REVIEW: 2020-06-29 Mon
:END:
<2020-06-16 Tue>
```

Then call `org-journal-new-entry`

**Expected behavior**
The task is carried over.

**What happens**
Emacs loops. I can `C-g` out of it, and this is what I see for today:

```org
* TODO Resserrer vélo elliptique
:PROPERT<2020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 Wed>020-06-17 WED>S:
:CREATED:  [2020-05-29 Fri 08:23]
:LAST_REVIEW: 2020-05-29 Fri
:NEXT_REVIEW: 2020-06-29 Mon
:END:
<2020-06-16 Tue>
```

**Comment by cslux** [comment]
@brabalan I can't reproduce it. It's carried over for me. Which version of org-journal do you have currently installed?

**Comment by brabalan** [comment]
I updated this morning. I tried again and it's still happening. I will try to debug it.

**Comment by cslux** [comment]
Call `toggle-debug-on-quit` call org-journal-new-entry and hit C-g when it hangs.

**Comment by brabalan** [comment]
I can reproduce with edebug.

The problem seems to happen in https://github.com/bastibe/org-journal/blob/master/org-journal.el#L796. The correct string is found, but the replacement is put in a completely different place (on the second line of the file on my case).

Could it be the two nested `save-excursion` do strange things, and how do you ensure the second `re-search-forward` (for the SCHEDULE) does not override the match data for the time stamp?

**Comment by cslux** [comment]
> `re-search-forward` (for the SCHEDULE) does not override the match data for the time stamp?

I use `save-match-data` the save the outermost match data.

**Comment by cslux** [comment]
Are you using daily journal files?

**Comment by cslux** [comment]
@brabalan Should be fixed with bb83eff

**Comment by brabalan** [comment]
Yes, I use daily files. And I see the bug now. (I was learning how to record a screen grab…)

Everything works, thanks!
